src/hotspot/cpu/x86/x86.ad

```diff
@@ -1750,7 +1750,6 @@ bool Matcher::match_rule_supported_vector(int opcode, int vlen, BasicType bt) {
         return false;
       }
       break;
-    case Op_ClearArray:
```

Comment:
This seems problematic, and may lead to the regression in https://bugs.openjdk.org/browse/JDK-8332487

On non-AVX512 platforms, this is now always `true` instead of always `false`. Probably this was not intended, and you thought this way going to be default `false`?

Reply:
Hi @eme64 , pattern predicates are checking the target features explicitly, this was introduced along with JDK-8257772, but looked redundant.

Reply:
I don't understand what you are implying. Are you saying this is not the reason for the regression?